### PR TITLE
more chains

### PR DESCRIPTION
**Pull request by mneko**

### CODE REVIEW

File: data/brands/shop/outdoor.json [hunk 1]
```diff
@@ -93,6 +93,18 @@
         "shop": "outdoor"
       }
     },
+    {
+      "displayName": "Fjällräven",
+      "id": "fjallraven-d45246",
+      "locationSet": {"include": ["001"]},
```
Comment (Identitaet) [inline data/brands/shop/outdoor.json:99]: they only seem to have their own stores in the US and Canada everywhere else it's partners

Comment (mneko) [reply data/brands/shop/outdoor.json]: i ran a build after changing it so it would be happy, and the build changed some other things, sorry

Comment (Identitaet) [reply data/brands/shop/outdoor.json]: the build looks to be correct and you fixed that small bit fast

**Comment by Identitaet** [comment]
Looks good but we should probably tell @bhousel about adding a new category of brands

**Comment by bhousel** [comment]
> Looks good but we should probably tell @bhousel about adding a new category of brands

`craft=electronics_repair` seems good to me 👍 
As long as it seems like a well-established tag, I'm ok with just about anything people are using.

**Pull request merged**